src/components/BookCard/BookCard.css

```diff
@@ -28,6 +28,7 @@
 .card-content {
   padding: 1.25em;
   margin-bottom: 0.625em;
+	
```

Comment:
We don't need a new line here. The next person who sees this is likely going to delete it in their MR.